<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog_Schleife_linear</class>
 <widget class="QDialog" name="Dialog_Schleife_linear">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>542</width>
    <height>425</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Schleife linear</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>380</y>
     <width>411</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="pushButton_abbrechen">
      <property name="text">
       <string>Abbrechen</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_speichern">
      <property name="text">
       <string>Speichern</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_ok">
      <property name="text">
       <string>OK</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>8</y>
     <width>521</width>
     <height>361</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="2" column="1">
     <widget class="QLabel" name="label_versatzx_var">
      <property name="text">
       <string>[VX]</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLabel" name="label_versatzy_var">
      <property name="text">
       <string>[VY]</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QSpinBox" name="spinBox_anzy">
      <property name="value">
       <number>1</number>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_versatzy">
      <property name="text">
       <string>Y-Versatz</string>
      </property>
     </widget>
    </item>
    <item row="3" column="2">
     <widget class="QLineEdit" name="lineEdit_versatzy"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_anzy">
      <property name="text">
       <string>Anz der Durchläufe Y</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLabel" name="label_anzy_var">
      <property name="text">
       <string>[AY]</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="label_anzx_var">
      <property name="text">
       <string>[AX]</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QSpinBox" name="spinBox_anzx">
      <property name="value">
       <number>1</number>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label_anzx">
      <property name="text">
       <string>Anz der Durchläufe X</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_versatzx">
      <property name="text">
       <string>X-Versatz</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_bez">
      <property name="text">
       <string>Bezeichnung</string>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QLineEdit" name="lineEdit_versatzx"/>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="label_afb">
      <property name="text">
       <string>Ausführbedingung</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QLabel" name="label_bez_var">
      <property name="text">
       <string>[BEZ]</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QLabel" name="label_afb_var">
      <property name="text">
       <string>[AFB]</string>
      </property>
     </widget>
    </item>
    <item row="4" column="2">
     <widget class="QLineEdit" name="lineEdit_bez"/>
    </item>
    <item row="5" column="2">
     <widget class="QLineEdit" name="lineEdit_afb"/>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>spinBox_anzx</tabstop>
  <tabstop>spinBox_anzy</tabstop>
  <tabstop>lineEdit_versatzx</tabstop>
  <tabstop>lineEdit_versatzy</tabstop>
  <tabstop>lineEdit_bez</tabstop>
  <tabstop>lineEdit_afb</tabstop>
  <tabstop>pushButton_abbrechen</tabstop>
  <tabstop>pushButton_speichern</tabstop>
  <tabstop>pushButton_ok</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
